lib/api.js

```diff
@@ -87,6 +87,21 @@ PlexAPI.prototype.postQuery = function query(options) {
     return this._request(options).then(uri.attach(url));
 };
 
+PlexAPI.prototype.putQuery = function query(options) {
```

Comment:
`function query(..)` should be `function putQuery(..)` here. It has impact on the potential stack traces generated.